shared-ui/src/androidMain/kotlin/co/touchlab/droidcon/ui/util/LocalImage.jvm.kt

```diff
@@ -19,6 +20,7 @@ import androidx.compose.ui.res.painterResource
 import co.touchlab.droidcon.ui.icons.Warning
 import co.touchlab.droidcon.ui.theme.Dimensions
 
+@SuppressLint("ComposableNaming", "DiscouragedApi")
```

Comment:
Comment on discouragedAPI, I think naming is obvious. (Is the non-standard naming required?)

Reply:
Tadeas came up with this name, it's just for internal use, the developer uses `LocalImage` which has the default values set. We can change the name of `__LocalImage` if you have an idea for a better name :)